config/manifests/inferencepool.yaml

```diff
@@ -75,6 +75,39 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 10
 ---
+apiVersion: gateway.envoyproxy.io/v1alpha1
```

Comment:
I moved this here into the inferencepool yaml because it references the epp service.